### PR TITLE
feat(entitytags): a simple `deleteByNamespace` admin api

### PR DESCRIPTION
**Pull request by ajordens**
Dry run (default):
```
curl -X POST 'https://localhost:8081/admin/tags/deleteByNamespace/{namespace}'
```

Remove from elasticsearch only:
```
curl -X POST 'https://localhost:8081/admin/tags/deleteByNamespace/{namespace}?dryRun=false'
```

Remove from elasticsearch and front50:
```
curl -X POST 'https://localhost:8081/admin/tags/deleteByNamespace/{namespace}?dryRun=false&deleteFromSource=true'
```

### CODE REVIEW

**Comment by ajordens** [comment]
Specifically opt'd not to make this an exposed API as I'm not quite sure how easy we'd like it to be to remove a whole lot of tags at one time. 

The logic is pretty straight forward, might warrant a unit test.

File: clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/model/ElasticSearchEntityTagsProvider.java [hunk 1]
```diff
@@ -417,6 +417,57 @@ public Map reconcile(String cloudProvider, String account, String region, boolea
     return getElasticSearchEntityTagsReconciler().reconcile(this, cloudProvider, account, region, dryRun);
   }
 
+  @Override
+  public Map<String, Object> deleteByNamespace(String namespace, boolean dryRun, boolean deleteFromSource) {
+    List<EntityTags> entityTagsForNamespace = front50Service
+      .getAllEntityTags(false)
```
Comment (ajordens) [inline clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/model/ElasticSearchEntityTagsProvider.java]: This just fetches the set of entity tags that front50 has in it's in-memory cache ... should be reasonable quick but the response payload is quite large (could be optimized in the future).

File: clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/model/ElasticSearchEntityTagsProvider.java [hunk 2]
```diff
@@ -417,6 +417,58 @@ public Map reconcile(String cloudProvider, String account, String region, boolea
...
+          .getTags()
+          .stream()
+          .filter(e -> !namespace.equalsIgnoreCase(e.getNamespace()))
+          .collect(Collectors.toList())
+      );
+
+      Set<String> tagNames = entityTags
+        .getTags()
+        .stream()
+        .map(e -> e.getName().toLowerCase())
```
Comment (christopherthielen) [inline clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/model/ElasticSearchEntityTagsProvider.java]: what is the reason for lowercase compare of the tag names?

**Comment by christopherthielen** [comment]
thanks for doing this!

**Pull request merged**